homework-slouthion/random-pic/app.js

```diff
@@ -0,0 +1,10 @@
+const randomBu = document.querySelector("#randomize");
+const changePic = function(){
+   console.log("You click me") ;
+   const newImage = document.createElement("img");
+    newImage.setAttribute("src", "https://source.unsplash.com/random")
```

Comment:
you can make the the bonus by getting the height and width as a variable and using them as a part of the URL

```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```